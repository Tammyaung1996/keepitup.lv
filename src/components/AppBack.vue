<script setup lang="ts">
const { router } = useAppRouter()
</script>

<template>
  <var-button color="transparent" text-color="#fff" round text @click="router.back">
    <var-icon class="text-[30px]!" name="chevron-left" />
  </var-button>
</template>
