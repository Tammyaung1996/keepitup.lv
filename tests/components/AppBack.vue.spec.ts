import AppBack from '@/components/AppBack.vue'
import { mount } from '@vue/test-utils'
import { expect, it } from 'vitest'

it('test AppBack mount', () => {
  const wrapper = mount(AppBack)

  expect(wrapper.html()).toMatchSnapshot()
})
