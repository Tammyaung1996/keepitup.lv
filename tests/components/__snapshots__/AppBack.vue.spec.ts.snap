// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`test AppBack mount 1`] = `
"<button class="var-button var--box var-button--normal var--inline-flex var-button--text var-button--text-default var-button--round" style="color: rgb(255, 255, 255); background: transparent;" type="button">
  <!--v-if-->
  <div class="var-button__content"><i class="var-icon var-icon--set var-icon-chevron-left text-[30px]!" style="transition-duration: 0ms;"></i></div>
  <div class="var-hover-overlay"></div>
</button>"
`;
